src/util/types.ts

```diff
@@ -1,45 +1,61 @@
 import * as semver from "semver";
 
-import { Inputs } from "../generated/inputs-outputs";
+// import { Inputs } from "../generated/inputs-outputs";
 
 // https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#template-literal-types
 // neat
-export type InstallableClient = `${Inputs}`;
+// export type InstallableClient = `${Inputs}`;
+export type InstallableClient = "crc" | "helm" | "kam" | "kamel" | "kn" | "oc" | "odo" | "openshift-install"
```

Comment:
Please suggest if there is any better way to achive this, basically I wanted to remove input `source` and `github_pat` from the inputs list.

Reply:
you can try the Omit or Exclude types https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
Great, Thanks!
`Exclude` worked.